--- freerdp-2.2.0/cmake/FindDocBookXSL.cmake.orig	2020-07-20 15:38:46.000000000 +0200
+++ freerdp-2.2.0/cmake/FindDocBookXSL.cmake	2020-11-02 22:00:52.583489912 +0100
@@ -13,7 +13,8 @@
 set (STYLESHEET_PATH_LIST
     share/xml/docbook/stylesheet/docbook-xsl
     share/xml/docbook/xsl-stylesheets
+    share/sgml/docbook/xsl-nons-stylesheets
     share/sgml/docbook/xsl-stylesheets
     share/xml/docbook/stylesheet/nwalsh/current
     share/xml/docbook/stylesheet/nwalsh
