--- FreeRDP/libfreerdp/codec/h264_ffmpeg.c~	2017-11-28 15:26:30.000000000 +0100
+++ FreeRDP/libfreerdp/codec/h264_ffmpeg.c	2018-04-29 09:50:33.774939164 +0200
@@ -146,7 +146,7 @@
 	};
 	av_opt_set(sys->codecEncoderContext, "preset", "veryfast", AV_OPT_SEARCH_CHILDREN);
 	av_opt_set(sys->codecEncoderContext, "tune", "zerolatency", AV_OPT_SEARCH_CHILDREN);
-	sys->codecEncoderContext->flags |= CODEC_FLAG_LOOP_FILTER;
+	sys->codecEncoderContext->flags |= AV_CODEC_FLAG_LOOP_FILTER;
 	sys->codecEncoderContext->me_cmp |= 1;
 	sys->codecEncoderContext->me_subpel_quality = 3;
 	sys->codecEncoderContext->me_range = 16;
@@ -404,9 +404,9 @@
 			goto EXCEPTION;
 		}
 
-		if (sys->codecDecoder->capabilities & CODEC_CAP_TRUNCATED)
+		if (sys->codecDecoder->capabilities & AV_CODEC_CAP_TRUNCATED)
 		{
-			sys->codecDecoderContext->flags |= CODEC_FLAG_TRUNCATED;
+			sys->codecDecoderContext->flags |= AV_CODEC_FLAG_TRUNCATED;
 		}
 
 		if (avcodec_open2(sys->codecDecoderContext, sys->codecDecoder, NULL) < 0)
--- FreeRDP/channels/tsmf/client/ffmpeg/tsmf_ffmpeg.c~	2017-11-28 15:26:30.000000000 +0100
+++ FreeRDP/channels/tsmf/client/ffmpeg/tsmf_ffmpeg.c	2018-04-29 09:51:28.065681976 +0200
@@ -207,8 +207,8 @@
 		}
 	}
 
-	if (mdecoder->codec->capabilities & CODEC_CAP_TRUNCATED)
-		mdecoder->codec_context->flags |= CODEC_FLAG_TRUNCATED;
+	if (mdecoder->codec->capabilities & AV_CODEC_CAP_TRUNCATED)
+		mdecoder->codec_context->flags |= AV_CODEC_FLAG_TRUNCATED;
 	return TRUE;
 }
 
