--- freerdp-3.5.1/CMakeLists.txt.orig	2024-04-22 11:18:39.000000000 +0200
+++ freerdp-3.5.1/CMakeLists.txt	2024-05-08 17:55:41.882600394 +0200
@@ -204,15 +204,6 @@ endif(EXPORT_ALL_SYMBOLS)
 
 # Compiler-specific flags
 if(CMAKE_COMPILER_IS_GNUCC)
-	if(CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64" OR CMAKE_SYSTEM_PROCESSOR MATCHES "i686")
-		CHECK_SYMBOL_EXISTS(__x86_64__ "" IS_X86_64)
-		if(IS_X86_64)
-			set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
-		else()
-			set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -march=i686")
-		endif()
-	endif()
-
 	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall")
 
 	if(NOT EXPORT_ALL_SYMBOLS)
