--- freerdp-3.9.0/winpr/libwinpr/sspi/Kerberos/kerberos.c.orig	2024-10-21 15:36:02.000000000 +0200
+++ freerdp-3.9.0/winpr/libwinpr/sspi/Kerberos/kerberos.c	2024-10-26 12:50:31.202989647 +0200
@@ -238,14 +238,14 @@ static BOOL isValidIP(const char* ipAddr
 	return isValidIPv4(ipAddress) || isValidIPv6(ipAddress);
 }
 
-static int build_krbtgt(krb5_context ctx, krb5_data* realm, krb5_principal* ptarget)
+static int build_krbtgt(krb5_context ctx, Realm* realm, krb5_principal* ptarget)
 {
 	/* "krbtgt/" + realm + "@" + realm */
 	size_t len = 0;
 	char* name = NULL;
 	krb5_error_code rv = KRB5_CC_NOMEM;
 
-	(void)winpr_asprintf(&name, &len, "krbtgt/%s@%s", realm->data, realm->data);
+	(void)winpr_asprintf(&name, &len, "krbtgt/%s@%s", realm, realm);
 	if (!name || (len == 0))
 		goto fail;
 
