--- FreeRDP-a132545545ef90534a64a08f5199741502de97e5/cmake/FindGSM.cmake.orig	2016-06-30 22:43:05.000000000 +0200
+++ FreeRDP-a132545545ef90534a64a08f5199741502de97e5/cmake/FindGSM.cmake	2016-07-02 20:05:44.567199213 +0200
@@ -1,4 +1,4 @@
-find_path(GSM_INCLUDE_DIR gsm/gsm.h)
+find_path(GSM_INCLUDE_DIR gsm.h)
 
 find_library(GSM_LIBRARY gsm)
 
--- a/libfreerdp/codec/dsp.c~	2018-08-09 10:41:56.000000000 +0200
+++ b/libfreerdp/codec/dsp.c	2018-08-12 14:19:24.735710945 +0200
@@ -31,7 +31,7 @@
 
 #if !defined(WITH_DSP_FFMPEG)
 #if defined(WITH_GSM)
-#include <gsm/gsm.h>
+#include <gsm.h>
 #endif
 
 #if defined(WITH_LAME)
